src/pages/mentees.tsx

```diff
@@ -47,6 +47,7 @@ import { formatMentorshipEndedAtText } from './mentees/[userId]';
 import { menteeAcceptanceYearField } from 'shared/menteeApplicationFields';
 
 const fixedFilter: UserFilter = { containsRoles: ["Mentee"] };
+type UpdateMenteeYear = (id: string, year: string) => void
```

Comment:
`id` => `userId`, `year` => `acceptanceYear`. Same for all occurrences of `id` and `year` in rest of the file.